src/systems/a320_systems/src/hydraulic/mod.rs

```diff
@@ -5202,7 +5204,10 @@ impl ElevatorAssembly {
               + ElectroHydrostaticPowered],
         current_pressure_outward: &impl SectionPressure,
         current_pressure_inward: &impl SectionPressure,
+        ths_actual_angle: Angle,
```

Comment:
Could make this trait implemented by `TrimmableHorizontalStabilizerAssembly` - but I don't feel strong about it.

Reply:
Done :)